### DISCUSSION TITLE
Amazon Machine Image Upgrade to Latest Version Doesn't work

### DISCUSSION THREAD

**Discussion by giesberge** [requarks/wiki#3987] [Error / Bug Report] [answered]
On my amazon machine image I get the following error
`Error: getaddrinfo ENOTFOUND wiki-update-companion`

**Comment by skledarg** [comment, marked as answer]
To update the wiki from the Administration panel, the wiki-update-companion container needs to be running. Use  `docker ps` to check that the wiki-update-companion is running. Do you use dockerized wikijs on your amazon machine? If so, check https://docs.requarks.io/install/ubuntu on how to set up the wiki-update-companion container.

**Reply by giesberge** [reply]
Amazon Machine Image should hypothetically all that up and running I chose that to be lazy

**Reply by Phil-hostwiki** [reply]
> Amazon Machine Image should hypothetically all that up and running I chose that to be lazy

If you are looking for a fully managed solution that takes care of updates, you can try https://hostwiki.io
